qiita_db/software.py

```diff
@@ -61,7 +65,10 @@ def get_commands_by_input_type(cls, artifact_types):
                         JOIN qiita.parameter_artifact_type
                             USING (command_parameter_id)
                         JOIN qiita.artifact_type USING (artifact_type_id)
+                        JOIN qiita.software_command USING (command_id)
                      WHERE artifact_type IN %s"""
+            if active_only:
+                sql = "%s AND active = True" % sql
```

Comment:
I almost thought there was a bug in there, was expecting something like `sql += ' AND active = True'`.